src/simple-cache-client.ts

```diff
@@ -500,7 +500,9 @@ export class SimpleCacheClient {
   public async dictionarySetFields(
     cacheName: string,
     dictionaryName: string,
-    items: Map<string | Uint8Array, string | Uint8Array>,
+    items:
+      | Map<string | Uint8Array, string | Uint8Array>
+      | Record<string, string | Uint8Array>,
```

Comment:
That's a mouthful. Could this be a type alias?

Reply:
sure, can do that in a followup PR